### PR TITLE
Extensions: Zoninator - Add DeleteZoneDialog to Edit Zone tab

### PR DESCRIPTION
**Pull request by ice9js**
This PR adds a confirmation dialog to the **Delete** button on Edit Zone tab in Zoninator extension.
The component/tab is not wired to the state yet - hence the temporary **[zone name]** placeholder.

screen shot 2017-08-31 at 12 29 37

# Testing

- Go to `/extensions/zoninator` and choose a site with Zoninator installed.
- Open the edit tab for one of the zones.
- Click **Delete** - the dialog should appear.
- Make sure you can close the dialog by clicking **Cancel** or clicking outside of it. No action is expected when clicking **Delete** on the dialog.

### CODE REVIEW

File: client/extensions/zoninator/components/settings/zone/index.jsx [hunk 1]
```diff
@@ -25,17 +26,35 @@ class Zone extends Component {
...
+					actionButton={ <Button compact primary scary onClick={ this.showDeleteDialog }>{ translate( 'Delete' ) }</Button> } >
 					{ translate( 'Edit zone' ) }
 				</HeaderCake>
 
+				{
+					showDeleteDialog &&
+					<DeleteZoneDialog
+						siteSlug={ siteSlug }
+						zoneName="[zone name]"
+						onConfirm={ noop }
```
Comment (donnapep) [inline client/extensions/zoninator/components/settings/zone/index.jsx]: So this `noop` is just a placeholder for now. Is that right?

Comment (ice9js) [reply client/extensions/zoninator/components/settings/zone/index.jsx]: That is correct :)

**Comment by rickybanister** [comment]
This should use the `is-primary` and `is-scary` property so the delete button becomes red. 

I also don't think it's necessary to mention the site name in the dialog since the picker should be setting that context. 

Everything looks great otherwise!

**Comment by folletto** [comment]
Post-merge note: please make sure in the future that there's a screenshot somewhere (ideally not updating the top, but in a comment, so history is preserved) so it's available for future reference and easy to identify changes. 

Thanks! :)

**Pull request merged**